specs/vep-2272-complete-data-job-configuration-persistence/README.md

```diff
@@ -128,6 +128,9 @@ replicated:
 
 * `git_commit_sha`
 * `vdk_version`
+* `vdk_image_name`
+* `image_name`
+* `cron_job_name`
```

Comment:
The code creates a name for the Cron job, and right now, that name is the same as the job's name. In other words, we don't need a separate column for that.